pyproject.toml

```diff
@@ -35,11 +36,13 @@ dependencies = [
   "textual>=3.4.0",
 ]
 
-[tool.setuptools.dynamic]
-version = { attr = "sot.__about__.__version__" }
+[tool.hatch.version]
+path = "src/sot/__about__.py"
 
-[tool.setuptools.packages.find]
-where = ["src"]
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "images",
+]
```

Comment:
The `images/` folder contains a GIF that is 3.1 MiB in size, which shows up in the source distribution because `hatchling` includes all files that are not excluded by the VCS by default. This means that the `tests/` folder is also included. There is some ambiguity about what should be included in sdists; some argue that they should contain `tests` and `docs`, while others disagree. I usually follow the suggestions or needs of the project maintainers, as there's no right answer. :)

These files are not included in the wheel, though, unless they are marked as such. The generated sdist is 30 KiB and the wheel is 28 KiB as a result.

Reply:
Sounds good! Addressed via cc7c01ce79f8f454ae3d241a6e733df7448afe19